src/identifiers.rs

```diff
@@ -53,6 +53,12 @@ impl UserId {
     pub fn to_string(&self) -> String {
         self.inner.as_str().to_owned()
     }
+
+    /// Clone this `UserId`.
```

Comment:
```suggestion
    /// Create a clone of this `UserId`.
    ///
    /// This can be useful when passing a UserId instance to methods such as {@link OlmMachine.updateTrackedUsers}
    /// which destroy the instance.